src/servicos/firestore.js

```diff
@@ -1,10 +1,10 @@
 import { db } from "../config/firebase";
 import { collection, addDoc, getDocs, doc, updateDoc, deleteDoc, query, onSnapshot } from "firebase/firestore"
 
-export async function salvarPost(data){
+export async function salvarPost(dados){
   try {
-    await addDoc(collection(db, 'posts'), data)
-    return 'ok'
+    const result = await addDoc(collection(db, 'posts'), dados)
```

Comment:
Esse result é uma referência do tipo result ou algo do tipo? Se não for o caso, acho que pode usar a tradução